changelogs/fragments/6199-archive-generate-checksum-in-chunks.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

I think this is more a bugfix than a feature.
```suggestion
bugfixes:
```